src/main/java/data_access/WeekDataAccess.java

```diff
@@ -41,14 +42,58 @@ public WeekDataAccess(ArrayList<Day> dataDays){
         days = dataDays;
     }
 
-    public ArrayList<Day> getDays() {
+    public static ArrayList<Day> getDays() {
         return days;
     }
 
     public static void setDays(ArrayList<Day> days) {
         WeekDataAccess.days = days;
     }
 
-    protected void save(){
+    /**
+     * Writes the contents of the days class variable onto a .ser file, persisting data
+     * @param days an ArrayList whose indices contain two hashmaps that contain events and tasks separately, representing the
+     *             week schedule
+     * @throws IOException
+     */
+    public static void writeObject(ArrayList<Day> days) throws IOException {
+        FileOutputStream fileOutputStream = new FileOutputStream("CleanCalendarStorage.txt");
+        ObjectOutputStream objectOutputStream = new ObjectOutputStream(fileOutputStream);
+        objectOutputStream.writeObject(new ArrayList<Day>(days));
+        objectOutputStream.flush();
+        objectOutputStream.close();
+    }
+
+    /**
+     * Reads the object from the .ser file and reinitialized day
+     * @throws ClassNotFoundException
+     * @throws IOException
+     */
+    public static void readObject() throws ClassNotFoundException, IOException {
```

Comment:
Again, why is this method public?

Reply:
This is called in the beginning of main, which is vital to load in days from previous uses of the program.